docs/environment_variables.md

```diff
@@ -132,6 +132,25 @@ Example:
 SSL_CERT_DIR=/path/to/ca-certs/ python -c "import httpx; httpx.get('https://example.com')"
 ```
 
+## `.NETRCFILE`
+
+Valid values: a filename
+
+If this environment variable is set and auth parameter is not defined, HTTPX will add auth information stored in
+.netrc file into request's header. If you do not provide NETRC environment, HTTPX will use default files. (~/.netrc
+, ~/_netrc)
```

Comment:
```suggestion
If this environment variable is set and auth parameter is not defined, HTTPX will add auth information stored in .netrc file into request's header. If you do not provide NETRC environment, HTTPX will use default files. (~/.netrc, ~/_netrc)
```

Reply:
Yep. Couldn't agree more. This seems far better than mine. Thank you for this kind suggestion and thoughtful modification. 🙏🏻

Reply:
Appreciating your style of interaction here, @meist0731. ✨